src/plone/app/theming/browser/controlpanel.pt

```diff
@@ -181,7 +181,7 @@
 
                             <form method="get" target="_blank" tal:attributes="action string:${context/absolute_url}/++theme++${theme/name}/@@download-zip">
                                 <button i18n:translate=""
-                                    class="btn btn-primary">Download</button>
+                                    class="btn btn-primary mt-1 px-3">Download</button>
```

Comment:
A class should not be applied to a single button that resides in a div that contains a group of buttons. Instead the [parent div of the group of buttons](https://github.com/plone/plone.app.theming/pull/213/files#diff-c96bcee60e1865b90e3c96e35ed68375a88a71130e599b8ec757dec76a5f0634R151) should have a class applied, specifically `d-grid gap-1`. See https://getbootstrap.com/docs/5.2/utilities/spacing/#gap.

```suggestion
                                    class="btn btn-primary">Download</button>
```

Reply:
actually, @stevepiercy I've done some slight changes to the block placement and don't need grid spacings
![image](https://user-images.githubusercontent.com/86848116/186424159-8b41a97c-df3e-4934-b1b3-f5f6708860c8.png)

Reply:
Those changes do not appear in this PR. Please commit and push the changes.

Reply:
@stevepiercy @mauritsvanrees  Committed changes... kindly have a look!